### PR TITLE
Removed `ListAssessmentResults` from assessment

### PR DESCRIPTION
**Pull request by oxisto**
This finally removes the `ListAssessmentResults` from the assessment service, which caused confusion. This also moves the `ListAssessmentResultsRequest` and response to the `orchestrator` package.

### CODE REVIEW

File: api/orchestrator/orchestrator.proto [hunk 1]
```diff
@@ -614,7 +613,7 @@ message AssessmentTool {
 
   // a list of metrics that this tool can assess, referred by their ids
   repeated string available_metrics = 4
-      [ (validate.rules).repeated.items.string.min_len = 1 ];
+      [ (validate.rules).repeated .items.string.min_len = 1 ];
```
Comment (anatheka) [inline api/orchestrator/orchestrator.proto:616]:
```suggestion
      [ (validate.rules).repeated.items.string.min_len = 1 ];
```

Comment (anatheka) [reply api/orchestrator/orchestrator.proto:616]: Why did you add the spaces behind .repeated?

File: service/assessment/assessment.go [hunk 2]
```diff
@@ -250,7 +243,7 @@ func (svc *Service) AssessEvidence(_ context.Context, req *assessment.AssessEvid
 	}
 
 	// Assess evidence
-	err = svc.handleEvidence(req.Evidence, resourceId)
+	_, err = svc.handleEvidence(req.Evidence, resourceId)
```
Comment (anatheka) [inline service/assessment/assessment.go:246]: Why are you retourning the assessment results in handleEvidence() if you not need them?

**Review by anatheka — COMMENTED** [review]
I'm not sure if it is a good idea to move the ListAssessmentResults to the orchestrator. This means we can no longer deploy the assessment component without the orchestrator. I am also not sure if we want to do that. 

Furthermore, if we leave it like this, shouldn't we also move the ListEvidences() and GetEvidence() to the Orchestrator?

File: service/assessment/assessment.go (hunk 2 above)
Comment (oxisto) [reply service/assessment/assessment.go:246]: We had one test that tested whether the results were created successfully. It kind of makes sense to return it, if somebody wants to have a look at it.

File: api/orchestrator/orchestrator.proto (hunk 1 above)
Comment (oxisto) [reply api/orchestrator/orchestrator.proto:616]: This is my stupid protobuf formatter, sorry.

File: service/assessment/assessment.go (hunk 2 above)
Comment (anatheka) [reply service/assessment/assessment.go:246]: You mean while debugging? That we can also do in the handle function, but ok.

**Review by anatheka — APPROVED** [review]

**Pull request merged**